demo/Titanic/README.md

```diff
@@ -45,6 +45,17 @@ To run the demo execute the `run_demo.py` script in either of the two available
 
 If you choose the `hit` scenario, don't be surprised if you encounter some failures along the route...
 
+### Step 4: Running the demo game (optional)
```

Comment:
This isn't really a step. Maybe we can refer to it as _Bonus: Play the demo as game_

Reply:
Will update